### ISSUE TITLE
[TF2] Class Icons are outdated and should be updated

### ISSUE THREAD

**Issue by Rongeno** [ValveSoftware/Source-1-Games#4178]
Currently the class icons for Pyro, Heavy, Engineer, Sniper and Spy are still using their beta design, while the others (Scout, Soldier, Demoman and Medic) look better and use the current designs, but overall the all look quite odd
![class icons](https://user-images.githubusercontent.com/108897156/184544923-ce2af0d9-8e32-45bf-8c82-e735e0609cde.jpg)

**Comment by AzureAzel** [comment]
Yeah, it's definitely a tad bit silly that a good portion of them are still using their beta designs. The same can be said with player gibs, but that'd likely have to be a separate issue.

**Comment by Whurrhurr** [comment]
actually looking at these class portraits, pyro, heavy, engie, sniper and spy all use the old beta lighting, but scout, soldier, demoman and medic use the current (at least as of 2007) lighting.

**Comment by condor00fr** [comment]
the portraits used in the hud for soldier, scout, demo ad medic are the same ones as the class selection screen, so why not apply the others to it?
